src/__tests__/city.test.js

```diff
@@ -0,0 +1,7 @@
+import city from '../__mocks__/city';
+
+describe('display missions component', () => {
+  test('snapshot renders properly', () => {
+    expect(city).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] All your tests are well written and passed.
<img width="348" alt="chrome_EAOxtYAQuP" src="https://user-images.githubusercontent.com/81468675/175512489-c7929748-9954-4bb0-89ee-3a31a842f9e3.png">